Cargo.toml

```diff
@@ -38,5 +38,6 @@ dtls = ["dep:webrtc-dtls", "dep:webrtc-util", "dep:rustls", "dep:rustls-pemfile"
 
 [dev-dependencies]
 quickcheck = "0.8.2"
-
+socket2 = "0.5"
+tokio-test = "0.4.4"
```

Comment:
These two crates seems not used in the code. Can they be deleted?

Reply:
They are used in the doc example that is tested. I can delete the example and the dependencies if that's preferable

Reply:
I see. I think it's fit in the dev-dependencies.